src/pages/generate.tsx

```diff
@@ -235,6 +236,7 @@ export default function ClientIdentifierGenerator() {
 
     const clientIdDocument = generateClientIdDocument({
       ...form.values,
+      contacts: form.values.contacts ? [form.values.contacts] : [],
```

Comment:
ditto

Reply:
resolved in https://github.com/inrupt/client-identifier-helper/pull/123